### ISSUE TITLE
audio buffer bug

### ISSUE THREAD

**Issue by n4hui** [cables-gl/cables_docs#51]
Hi! The audiobuffer player does not stop even with the loop option unchecked. Here is the example patch playing an audio,

https://cables.gl/p/610cc0402bd9067c51c5396a

I test this in chrome.

**Comment by tastez** [comment]
when you test this patch on your setup, does the sound start playing right away?
what happens in another browser/computer?

here the patch is silent and will only play the sound if you untoggle and toggle the sidebar. Once it is playing it stops when untoggling.

 Due to permissions, auto playback of audio is blocked - you must use the playbutton op in your patch in most cases to get user interaction first.

could you please fill out the entire bug report template in the future, it helps a lot!
please let me know your chrome version and OS version just in case.

**Comment by n4hui** [comment]
yes, we are working in an online team with a patch with this issue since the new cables update, i added the comment in the edit mode of the patch, before it played the sound, but stopped at the end, then you triggered again to replay. Now it has a permanent loop, so i need it to stop, because thats the trigger to my other patch initialize. Thank you in advance!
